PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -5,4 +5,4 @@ intellij_version=IC-2020.2
 dep_plugins=org.intellij.scala:2020.2.14
 applicationinsights.key=57cc111a-36a8-44b3-b044-25d293b8b77c
 updateVersionRange=true
-patchPluginXmlSinceBuild=202.6397.20
\ No newline at end of file
+patchPluginXmlSinceBuild=202.6397.94
```

Comment:
Would the minor version have to change?

Reply:
Should be fine, refer to this: https://confluence.jetbrains.com/display/IDEADEV/IntelliJ+IDEA+2020.2+%28202.6397.94+build%29+Release+Notes

202.6397.94 is 202 official build number.